subliminal/subtitle.py

```diff
@@ -276,13 +275,16 @@ def guess_matches(video, guess, partial=False):
     matches = set()
     if isinstance(video, Episode):
         # series
-        if video.series and 'series' in guess and sanitized_string_equal(guess['series'], video.series):
+        if video.series and 'title' in guess and sanitized_string_equal(guess['title'], video.series):
             matches.add('series')
+        # series
+        if video.title and 'episode_title' in guess and sanitized_string_equal(guess['episode_title'], video.title):
+            matches.add('title')
```

Comment:
Just my two cents worth, I find the series/title a better format too and it still works for movies as the "series" title for a movie can be considered the set title, e.g. `Back to the Future Trilogy` and `Back to the Future Part III` however i can see how this could potentially complicate the code unnecessarily.